src/confd/src/infix-services.c

```diff
@@ -307,6 +386,128 @@ static int web_change(sr_session_ctx_t *session, uint32_t sub_id, const char *mo
 	return put(cfg, srv);
 }
 
+/**
+ * Recursive directory remove (rm -r)
+ * @param file or directory to remove
+ *
+ * @returns 0 on success, else error
+ */
+
+int rm(const char *path) {
+    struct dirent *entry;
+    struct stat statbuf;
+    DIR *dir;
+    if (!path) {
+	    errno = EINVAL;
+	    return 1;
+    }
```

Comment:
Same here, add missing whitespace and make sure to have start brace on a new line for functions :-)